quipucords/conftest.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2022  Red Hat, Inc.
+
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+
+"""
+pytest config for quipucords.
+
+IMPORTANT: Don't import aplication code on toplevel as this can influence
+the proper patching.
+"""
+# pylint: disable=import-outside-toplevel
+from contextlib import suppress
+from unittest.mock import patch
+
+import pytest
+
+
+def _kill_scan_manager(manager_instance):
+    for job in manager_instance.scan_queue:
+        with suppress(AttributeError):
```

Comment:
Is there a difference between using contextlib.suppress and a try/catch besides readability?

Reply:
nope. this is exactly the same as 
```
try:
   job.kill()
except AttributeError:
    pass
```
just less verbose